apps/admin-gui-e2e/src/integration/resource-management-perun_admin.spec.js

```diff
@@ -0,0 +1,150 @@
+context('Actions', () => {
+  const dbVoName = 'test-e2e-vo-from-db-3';
+  const dbResourceName = 'test-e2e-resource-from-db-2';
+
+  const dbAddManager = 'test4';
+  const dbRemoveManager = 'resource-admin-2';
+
+  const dbAttributeToAdd = 'user-settings-name';
+  const dbAttributeToDelete = 'user-settings-description';
+
+  const dbGroupToAssign = 'test-e2e-group-from-db-2';
+  const dbGroupToRemove = 'test-e2e-group-from-db-1';
+
+  before(() => {
+    if (Cypress.env('BA_USERNAME_RESOURCE_MANAGER')) {
+      sessionStorage.setItem('baPrincipal', '{"name": "resourceManager"}');
+      sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME_RESOURCE_MANAGER'));
+      sessionStorage.setItem('basicPassword', Cypress.env('BA_PASSWORD_RESOURCE_MANAGER'));
+      cy.visit('service-access');
+    }
+  });
+
+  beforeEach(() => {
+    cy.visit('organizations')
+      .get(`[data-cy=${dbVoName}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get('[data-cy=resource-list]')
+      .click()
+      .get(`[data-cy=${dbResourceName}]`)
+      .click()
+  })
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .get('[data-cy=attributes]')
+      .click()
+      .get('[data-cy=add-attributes]')
+      .click()
+      .get(`[data-cy=${dbAttributeToAdd}-value]`)
+      .type('test')
+      .intercept('**/attributesManager/getAttributes/vo**').as('getAttributes')
+      .get('[data-cy=save-selected-attributes]')
+      .click()
```

Comment:
You want to use the `intercept()` method because you want to somehow react to the interception further when you detect some response. In our e2e tests, we always want to wait for the response and don't check the UI before the response will come (otherwise you can check the incorrect UI - dialog instead of the final page of this test).
```suggestion
      .click()
      .wait('@getAttributes')
```
![image](https://user-images.githubusercontent.com/70371638/179741760-26eb743b-6169-4f92-87d1-6c040cfe31df.png)
![image](https://user-images.githubusercontent.com/70371638/179742208-82c2e2b0-4936-44fe-95da-e33d619e0751.png)